### ISSUE TITLE
Add doc to explain scenario of how clusterpedia can help

### ISSUE THREAD

**Issue by panpan0000** [clusterpedia-io/clusterpedia#109]
### What would you like to be added?

The `Features List` can not directly make people to adopt it.

> `To a man with a hammer, everything looks like a nail.`

Now we have the hammer, but how to promote to the world more effectively , so those nails can find this hammer?

1. SOE maybe one way. Maybe you can  describe as many as possible scenario  ,which this hammer can help .
2. Treat the project as a product, do everything those product manager will do : analysis and advocation ..etc

Just my two cents. 
The only "nail" I can figure out is the tools acting as cluster-gateway like PaaS. But I believe this hammer could fit more nail. 

### Why is this needed?

as above.

**Comment by Iceber** [comment]
@panpan0000 Yes, I should indeed reflect and summarize my recent work. In fact, a lot of what I'm doing now is just making clusterpedia even better than it already is, for example with small features and additional unit tests, which are important, but they don't attract more people to use and participate in it.

The real attraction for others is to show how clusterpedia fits in well with other products(eg. karmada, clusternet ...) and gives users useful features that other products don't have.

There is also a need to consider how to make greater use of the multi-cluster resources collected, in addition to providing the ability to search for resources, to provide more powerful capabilities.

I probably need to focus my energy and time mainly on what would make clusterpedia more appealing.
